chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt

```diff
@@ -265,28 +264,16 @@ object InstanceActions {
         )
     }.withName("Create new user: $id")
 
-    fun changeRoomName(room: Room, newName: String): InstanceAction = {
-        chatkitInstance.request<JsonElement>(
-                options = RequestOptions(
-                        path = "/rooms/${room.id}",
-                        method = "PUT",
-                        body = mapOf("name" to newName).toJson()
-                ),
-                tokenProvider = sudoTokenProvider,
-                responseParser = { it.parseAs() }
-        )
-    }.withName("Changing name of room ${room.name} to $newName ")
-
-    fun deleteRoom(room: Room): InstanceAction = {
+    fun deleteRoom(roomId: String): InstanceAction = {
```

Comment:
I think we should remove this method as it's better to test SDK around this case with a functional level test that fakes the backend.

Reply:
👍 done.